src/lifecycle/properties-ready.js

```diff
@@ -0,0 +1,6 @@
+export default function propertiesApply (elem, props) {
+  Object.keys(props).forEach(function (name) {
+    let prop = props[name];
+    prop.set && prop.set.call(elem, elem[name]);
```

Comment:
Not necessarily. If you were to define your own property function that returns a definition, you don't have to define `set`. This is just defensive of that possibility.